config/jobs/kubernetes-sigs/sig-windows/soak-tests.yaml

```diff
@@ -7,56 +24,83 @@ presubmits:
     optional: true
     decoration_config:
       timeout: 8h
-    path_alias: k8s.io/perf-tests
     labels:
       preset-service-account: "true"
       preset-dind-enabled: "true"
       preset-kind-volume-mounts: "true"
       preset-azure-cred-only: "true"
       preset-azure-anonymous-pull: "true"
-    extra_refs:
-      # This needs to be uncommented when we switch to periodic job.
-      # - org: kubernetes
-      #   repo: perf-tests
-      #   base_ref: "main"
-      #   path_alias: "k8s.io/perf-tests"
-      - org: kubernetes-sigs
-        repo: cluster-api-provider-azure
-        base_ref: release-1.6
-        path_alias: "sigs.k8s.io/cluster-api-provider-azure"
-        workdir: true
+      preset-azure-win-soak-test-creds: "true"
     spec:
       containers:
         - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230111-cd1b3caf9c-master
           command:
-            - runner.sh
-            - ./scripts/ci-entrypoint.sh
+            - ./run-e2e.sh
```

Comment:
@oprinmarius I think these settings need to change too?  In the case of running form `perf-tests` repo we need to create the cluster from scratch?

Reply:
@jsturtevant if we want to build a new cluster from scratch then yes, we need to create a presubmit job similar to periodic-soak-tests-capz-windows-2019 . 
Currently is reusing the win-soak-test cluster.

Reply:
Yes, I think we would on a pre-submit as to not interfere with any other tests running on the cluster at the same time.

Reply:
Makes sense. I pushed the new changes